components/AlertNotification/AlertNotification.tsx

```diff
@@ -46,6 +48,16 @@ const Alert = ({
       <Icon className="h-4 w-4" data-testid="alert-icon" />
       <AlertTitle title={title} data-testid="alert-title" />
       <AlertDescription message={message} data-testid="alert-message" />
+      <Button
+        variant="ghost"
+        size="icon"
+        className="absolute right-4 top-2"
+        data-testid="dismiss-alert-btn"
```

Comment:
nit: I don't see an associated test that utilizes this `data-testid` attribute. Do we need this here, or is it an indicator that a test is coming? Regardless, this button _could_ theoretically be found/queried in the test without this attribute.

Reply:
@braydoncoyer I haven't created a test for this yet. Created my PR too soon!

Reply:
@braydoncoyer tests have been created and this PR is ready for review!

Reply:
> nit: I don't see an associated test that utilizes this `data-testid` attribute. Do we need this here, or is it an indicator that a test is coming? Regardless, this button _could_ theoretically be found/queried in the test without this attribute.

If there's an element that needs to be tested, we use the `data-testid` attribute. This shows clear indication that element is being tested by a unit test.